sdks/go/pkg/beam/runners/direct/direct.go

```diff
@@ -74,10 +74,27 @@ func Execute(ctx context.Context, p *beam.Pipeline) (beam.PipelineResult, error)
 	if err = plan.Down(ctx); err != nil {
 		return nil, err
 	}
-	// TODO(lostluck) 2020/01/24: What's the right way to expose the
-	// metrics store for the direct runner?
-	metrics.DumpToLog(ctx)
-	return nil, nil
+
+	prResult, prerr := newDirectPipelineResult(ctx)
+	return prResult, prerr
```

Comment:
This should be able to inlined with the return.

```suggestion
	return newDirectPipelineResult(ctx)
```